### PR TITLE
chore: bump up templates' sdk ver to 1.0.0

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: templates/tab/ts/default/package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
         "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.3511fba",
         "@microsoft/microsoft-graph-client": "^3.0.1",
         "@microsoft/teams-js": "^1.9.0",
-        "@microsoft/teamsfx": "^0.6.0",
-        "@microsoft/teamsfx-react": "0.1.0",
+        "@microsoft/teamsfx": "1.0.0-rc.0",
+        "@microsoft/teamsfx-react": "1.0.0",
```
Comment (tecton) [inline templates/tab/ts/default/package.json:13]: Missing rc here?

Comment (wenytang-ms) [reply templates/tab/ts/default/package.json]: updated.

**Review by tecton — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

File: templates/tab/js/m365/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "@microsoft/mgt-teamsfx-provider": "2.4.1-next.teamsfx.1552a26",
     "@microsoft/microsoft-graph-client": "^3.0.1",
     "@microsoft/teams-js": "2.0.0-beta.6",
-    "@microsoft/teamsfx": "0.7.0-beta.0",
+    "@microsoft/teamsfx": "1.0.0-rc.0",
```
Comment (kuojianlu) [inline templates/tab/js/m365/package.json:12]: No need to update `teamsfx` version to `1.0.0-rc.0` for M365 templates. Once `teams-js@2.0.0` is released, we will release `teamsfx@2.0.0-beta.0` manually, and then update the `teams-js` and `teamsfx` version.

Comment (wenytang-ms) [reply templates/tab/js/m365/package.json]: sure, revert this change

**Review by kuojianlu — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**